inst/crunch-test.R

```diff
@@ -63,7 +56,7 @@ silencer <- temp.option(show.error.messages=FALSE)
 
 assign("entities.created", c(), envir=globalenv())
 with_test_authentication <- function (expr) {
-    if (run.integration.tests) {
+    if (getOption("run.integration.tests")) {
```

Comment:
Why this change?

Reply:
It was necessary for another idea I had, but I decided not to implement that. I kept it because it's more consistent with the other options we are using to control what's going on with the tests (`test.user` and `test.pw`). On a much more minor point: It also keeps the integration setting in a harder to read place than a standard object is, so it's a bit harder to accidentally over-write `run.integration.tests` without actively setting an object. Though I admit it is very unlikely that any developer would do this. I don't feel strongly about it either way, I'm happy to change it back to be a global object.

Reply:
I'm not worried about a user overwriting `run.integration.tests`. I don't really think it's worth moving this to an option; it's not really more consistent because you aren't setting it with `envOrOption`, and it's naming isn't consistent with the other args (namespace, R_ prefix, etc.). And I don't think that changing all that to make it fully consistent with the other options is worth our time.